test/instrumentation/modules/hapi.js

```diff
@@ -14,7 +14,7 @@ var semver = require('semver')
 if (semver.lt(process.version, '8.9.0') && semver.gte(pkg.version, '17.0.0')) process.exit()
 
 // hapi does not work on Node.js 10 because of https://github.com/nodejs/node/issues/20516
-if (semver.gte(process.version, '10.0.0-rc')) process.exit()
+if (semver.satisfies(process.version, '>= 10.0.0-rc < 10.8')) process.exit()
```

Comment:
I think it's more readable if it's written in the "official" way without the whitespace, i.e. `>=10.0.0-rc <10.8`

Reply:
fixed